parm/stage/atmosphere_warm_RT.yaml.j2

```
@@ -0,0 +1,51 @@
+atmosphere_warm_RT:
+    mkdir:
+        {% for mem in range(first_mem, last_mem + 1) %}
+        {% set COMOUT_ATMOS_RESTART_PREV_MEM = COMOUT_ATMOS_RESTART_PREV_MEM_list[mem] %}
+        - "{{ COMOUT_ATMOS_RESTART_PREV_MEM }}"
+        {% set COMOUT_ATMOS_ANALYSIS_MEM = COMOUT_ATMOS_ANALYSIS_MEM_list[mem] %}
+        - "{{ COMOUT_ATMOS_ANALYSIS_MEM }}"
+        {% endfor %} # mem loop
+    copy:
+        #select restart members files (80) specific to the cycle
+        {% set cyc_ranges = ({ "00": [(range(1, 31), 0)],
+                               "06": [(range(21, 51), 20)],
+                               "12": [(range(41, 71), 40)],
+                               "18": [(range(61, 81), 60), (range(1, 11), -20)]}) %}
```

Having the offsets in this array is a little confusing.  I think there may be a way to calculate the indexes on the fly.  Please at least split this array into `cyc_ranges` and `cyc_offsets`.  Also, you can concatenate ranges by first converting them to lists.

```suggestion
        {% set cyc_ranges = { "00": list(range(1, 31)),
                               "06": list(range(21, 51)),
                               "12": list(range(41, 71)),
                               "18": list(range(61, 81)) + list(range(1, 11)} %}

        {% set cyc_offsets = { "00": [0],
                               "06": [20],
                               "12": [40],
                               "18": [60, -20]} %}